website/testCSSOrder.mjs

```diff
@@ -49,6 +48,7 @@ const EXPECTED_CSS_MARKERS = [
 
   // Lazy-loaded lib
   '.DocSearch-Modal',
+  '.DocSearch-Hit-content-wrapper',
```

Comment:
I'm surprised that this moved, but it makes more sense this way? Not sure why it was loaded so early before.